static/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -267,13 +268,15 @@ class Actions extends React.Component<Props, State> {
         >
           <ReviewAction onUpdate={this.onUpdate} disabled={!group.inbox || disabled} />
         </Tooltip>
-        <DeleteAction
-          disabled={disabled}
-          organization={organization}
-          project={project}
-          onDelete={this.onDelete}
-          onDiscard={this.onDiscard}
-        />
+        <Access organization={organization} access={['event:admin']}>
```

Comment:
this looks good, but i think we want to use the result of the access component to disable the button rather than removing it.

Kinda like how this one works https://github.com/getsentry/sentry/blob/1eb8781906e585b20ff0a72e537d30eba2cf507b/static/app/components/integrationExternalMappings.tsx#L224-L225